<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1677215</width>
       <height>1667215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background-color: rgb(85, 85, 255);
border:6px solid grey;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
border:none;
padding:10px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Team meeting_Monochromatic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_8">
        <property name="styleSheet">
         <string notr="true">QFrame{background-color: none;
padding:20px;
padding-bottom:20px;
border:none;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowminimumheight="0,0,0,0,0,1,1,1,1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="7" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
border-radius:8px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
background-color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;
border-radius:8px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;
background-color: rgb(255, 255, 255);

</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
margin-bottom:0px;
border-radius:8px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
background-color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
border-radius:8px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
background-color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_9">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton:hover{

	color: rgb(255, 255, 255);
}
QPushButton{

font: 87 14pt &quot;Arial Black&quot;;

color: rgb(255, 255, 255);
border-radius:0px;
background:none;
margin-right:5px;


}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../contra_new/Icons/minimizeicon.png</normaloff>../contra_new/Icons/minimizeicon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>42</width>
             <height>42</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;
border:none;
border-radius:8px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;
background-color: rgb(255, 255, 255);

</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>Username</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 87 24pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
padding:0;
border:none;
background:none;
</string>
           </property>
           <property name="text">
            <string>New User !</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
border-radius:8px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
background-color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;
border-radius:8px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;
background-color: rgb(255, 255, 255);

</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>Confirm Password</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_8">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton:hover{

	color: rgb(255, 255, 255);
}
QPushButton{

font: 87 14pt &quot;Arial Black&quot;;

color: rgb(255, 255, 255);
border-radius:0px;
background:none;


}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../contra_new/Icons/closeicon.png</normaloff>../contra_new/Icons/closeicon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;
border-radius:8px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;
background-color: rgb(255, 255, 255);

</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string>Master Password</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="4">
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>65</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(85, 170, 255);
	
	background-color: rgb(255, 0, 127);
border-radius:8px;

padding:0px;

font: 87 10pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
border-radius5px;}
QPushButton:hover{
	
	background-color: rgb(85, 170, 255);
}</string>
           </property>
           <property name="text">
            <string>ADD USER</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_9</tabstop>
 </tabstops>
 <resources>
  <include location="../FINACKO/resources.qrc"/>
 </resources>
 <connections/>
</ui>
